skbio/metadata/tests/test_intersection.py

```diff
@@ -0,0 +1,226 @@
+# ----------------------------------------------------------------------------
```

Comment:
I don't think we can change the copyright of the file. I know you had to add it because checklist.py will complain otherwise. I'll issue a PR shortly to add a directive to checklist.py allowing certain files to be skipped. I'll follow up here when that's merged and provide instructions on how to ignore this file.